src/SignalRServiceExtension/TriggerBindings/NullListener.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.Listeners;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    internal class NullListener: IListener
```

Comment:
Does Microsoft.Azure.WebJobs.Host.Listeners; contains NullListeners so we can remove this class

Reply:
No, every extensions which need `NullListener` defined there own.